File: Day 60/homework/codewars 2/answer.py
# https://www.codewars.com/kata/52b757663a95b11b3d00062d/train/python

def to_weird_case(string):
    ans = ''
    i = True
    for char in string:
        if i == True:
            ans += char.upper()
            i = False
        elif i == False:
            ans += char.lower()
            i = True
        if char == " ":
            ans += ""
            i = True
    return ans

File: Day 063/homework/codewars/answer.js
// https://www.codewars.com/kata/53dbd5315a3c69eed20002dd/train/javascript

function filter_list(l) {
    let ans = [];
    for (let num of l) {
      if (Number.isInteger(num)) {
        ans.push(num);
      }
    }
    return ans;
  }